qt/scientific_interfaces/ISISReflectometry/CMakeLists.txt

```diff
@@ -1,100 +1,80 @@
+FUNCTION(PREPEND var prefix)
+   SET(listVar "")
+   FOREACH(f ${ARGN})
+      LIST(APPEND listVar "${prefix}/${f}")
+   ENDFOREACH(f)
+   SET(${var} ${listVar} PARENT_SCOPE)
+ENDFUNCTION(PREPEND)
+
+add_subdirectory(GUI)
+add_subdirectory(Reduction)
+
+message(${GUI_SRC_FILES})
```

Comment:
As this is going to an integration branch I won't force this to go back round again but could we make a note to remove this print statement before it gets to master. The CMake output is already quite crowded.